libs/go/ztsroletoken/role-token.go

```diff
@@ -0,0 +1,177 @@
+package roletoken
```

Comment:
add license lines:

// Copyright 2017 Yahoo Holdings, Inc.
// Licensed under the terms of the Apache version 2.0 license. See LICENSE file for terms.